### DISCUSSION TITLE
Use Google Generative AI to describe the image

### DISCUSSION THREAD

**Discussion by wild-a** [koush/scrypted#1364] [Ideas]
[Google Generative AI](https://www.home-assistant.io/integrations/google_generative_ai_conversation/?fbclid=IwAR2Mz5ujGlqJOt00EEGZE_ZoolHGaYegNmcOipB9z9Edl0tM8PtBaMa1mGg_aem_AQUExwe4J3no_fUQ5I_fuyRQl1KcK85sCIIfeLGUUVDy0MHasoQyo5FrwIpq89vylQA) can be used in Home Assistant to describe motion-triggered snapshots. Can it be used with Scrypted?

**Comment by wild-a** [comment]
@koush Any thoughts?

**Comment by rooi** [comment]
Hi,

I found the following AI capabilities and erupted integration may allow watching video in real time to enable smart notification and logging:
https://github.com/cocktailpeanut/mirror

This should allow prompts like:

What is this object I am holding?
What is the person doing?
Describe some notable events in the image.
How many people are in this picture?
Let me know if you see anything weird.

If this works as demonstrated, integration in scripted would perhaps be of added value. For example, for smart logging events, or to notify on specific questions (e.g. for my situation, notify me when a parking spot becomes available)

Let me know what you think. Perhaps something on the long term roadmap?

Best,

Roy

**Reply by sfnemis** [reply]
https://firat.nemis.me/scrypted-and-home-assistant-google-gemini-ai-integration-de1828869b04
You can check this

**Comment by sfnemis** [comment]
> [Google Generative AI](https://www.home-assistant.io/integrations/google_generative_ai_conversation/?fbclid=IwAR2Mz5ujGlqJOt00EEGZE_ZoolHGaYegNmcOipB9z9Edl0tM8PtBaMa1mGg_aem_AQUExwe4J3no_fUQ5I_fuyRQl1KcK85sCIIfeLGUUVDy0MHasoQyo5FrwIpq89vylQA) can be used in Home Assistant to describe motion-triggered snapshots. Can it be used with Scrypted?

https://firat.nemis.me/scrypted-and-home-assistant-google-gemini-ai-integration-de1828869b04

**Reply by wild-a** [reply]
This worked perfectly. Thanks!

**Reply by wild-a** [reply]
Just to add additional information, the integration into home assistant via MQTT is great, took me less than 30 minutes to get the whole thing set up and running.
